droidlet/dashboard/web/src/components/Memory2D.js

```diff
@@ -275,6 +275,7 @@ class Memory2D extends React.Component {
             fill={color}
             onClick={(e) => {
               this.handleObjClick("obstacle_map", x, y, {
+                memid: "don't edit",
```

Comment:
These objects refer to those obtained from `get_obstacle_list` defined in `droidlet/memory/place_field`. I believe some examples of these objects are "hole" or "pit" that are initialized with the world. I don't know how we should deal with these on the map (they aren't drawn from ReferenceObjects), so I left this "don't edit" placeholder